ci/helm-chart/Chart.yaml

```diff
@@ -20,4 +20,4 @@ version: 2.2.0
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 4.1.0
+appVersion: 4.2.0
```

Comment:
I think we are supposed to bump `version` in this file as well.

Reply:
ah! that must not be part of the release prep script. nice catch!